prow/config/tide.yaml

```diff
@@ -20,21 +20,11 @@ tide:
 
   - repos:
     - maistra/test-infra
```

Comment:
I think we can move the test-infra jobs to openshift CI too. Please, create a ticket for it if we don't have it yet.

We can start simply by configuring tide for the test-infra repo. Thus we can eliminate this file completely.

Reply:
okay, new ticket : https://issues.redhat.com/browse/OSSM-3666